discovery-provider/ddl/migrations/0032_add_usdc_purchases_extra_amount.sql

```diff
@@ -0,0 +1,8 @@
+begin;
+
+ALTER TABLE
+    usdc_purchases
+ADD
+    COLUMN IF NOT EXISTS extra_amount BIGINT NOT NULL DEFAULT 0;
```

Comment:
Open to feedback on this. I don't see us default initializing many columns, especially numbers. If it makes more sense to let this be nullable, I can try to rework it to correctly support that.

Reply:
I think this is fine. One super nit: is this cents or wei? if cents then normal `INT` type is probably sufficient? 21mil is a lot to pay extra

Reply:
It's wei. I wanted to keep it consistent with the `amount` column. Both use bigint because they are wei values, even though practically speaking they should never be so huge that an int would be too small. It's good signaling that this isn't a "safe" value to throw into a number.